synapse/tests/test_tools_csvtool.py

```diff
@@ -33,7 +33,7 @@
 
 # count is used for test coverage.
 csvstorm_export = b'''
-test:int $lib.csv.emit($node, :loc) | count
+test:int $lib.csv.emit($node, $node.props.loc) | count
```

Comment:
was this a required or demonstrative change?

Reply:
again, required due to not all nodes having the `:loc` prop